### PR TITLE
Update accounting.formatMoney definition to allow string input

### PR DESCRIPTION
**Pull request by chris-codaio**
- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/openexchangerates/accounting.js/blob/master/accounting.js#L287
- [X] Increase the version number in the header if appropriate.

### CODE REVIEW

**Comment by chrisleck** [comment]
@gerich-home, please take a look.

File: types/accounting/accounting-tests.ts [hunk 1]
```diff
@@ -3,6 +3,9 @@
 // Default usage:
 accounting.formatMoney(12345678); // $12,345,678.00
 
+// Stringified usage:
+accounting.formatMoney('$4394958309392.9401'); // $4,394,958,309,392.94
```
Comment (gerich-home) [inline types/accounting/accounting-tests.ts]: Could you please show me such example in their documentation/code?
I was not able to find an example in their documentation: http://openexchangerates.github.io/accounting.js/

File: types/accounting/index.d.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
-// Type definitions for accounting.js 0.3
+// Type definitions for accounting.js 0.4
 // Project: http://josscrowcroft.github.io/accounting.js/
```
Comment (gerich-home) [inline types/accounting/index.d.ts]: The site is dead now, probably we also need to update the link to the http://openexchangerates.github.io/accounting.js/

**Review by gerich-home — CHANGES_REQUESTED** [review]

File: types/accounting/accounting-tests.ts (hunk 1 above)
Comment (chrisleck) [inline types/accounting/accounting-tests.ts]: If you follow the link I give above, you see the string parsing code under `formatMoney` via `unformat`.  We use this successfully in our (private) codebase.

File: types/accounting/index.d.ts (hunk 2 above)
Comment (chrisleck) [inline types/accounting/index.d.ts]: Updated.

**Comment by chrisleck** [comment]
@gerich-home please take another look.

**Review by gerich-home — APPROVED** [review]
Looks good to me

**Comment by chrisleck** [comment]
Thanks!  Not sure how to actually get this committed now :)

**Pull request merged**